src/components/search/sort-box.tsx

```diff
@@ -15,7 +15,10 @@ const SortBox: React.FunctionComponent<SortBoxProps> = ({
         }}
         label="Sort by"
         name="sort-by-dropdown"
-        options={Object.keys(sortByOptions)}
+        // 'Closest Upcoming' is reserved for events.
+        options={Object.keys(sortByOptions).filter(
+            mode => mode !== 'Closest Upcoming'
+        )}
```

Comment:
This is a fix to a bug I introduced when adding `Closest Upcoming` as a sort mode for events. Didn't realize it was being used here.